### PR TITLE
[4.0] com_users sidebar

### PR DESCRIPTION
**Pull request by brianteeman**
Removes the unused sidebar code from com_users

### CODE REVIEW

File: administrator/components/com_users/View/Notes/HtmlView.php [hunk 1]
```diff
@@ -185,7 +176,6 @@ protected function addToolbar()
 
 		$toolbar->help('JHELP_USERS_USER_NOTES');
```
Comment (Quy) [inline administrator/components/com_users/View/Notes/HtmlView.php]:
```suggestion
```

File: administrator/components/com_users/tmpl/groups/default.php [hunk 2]
```diff
@@ -28,12 +28,7 @@
 ?>
 <form action="<?php echo Route::_('index.php?option=com_users&view=groups'); ?>" method="post" name="adminForm" id="adminForm">
 	<div class="row">
-		<?php if (!empty($this->sidebar)) : ?>
-            <div id="j-sidebar-container" class="col-md-2">
-				<?php echo $this->sidebar; ?>
-            </div>
-		<?php endif; ?>
-        <div class="<?php if (!empty($this->sidebar)) {echo 'col-md-10'; } else { echo 'col-md-12'; } ?>">
+        <div class="col-md-12">
```
Comment (Quy) [inline administrator/components/com_users/tmpl/groups/default.php]:
```suggestion
		<div class="col-md-12">
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b3fdbeb7d1624458b52ac573e3924f13f6745bd1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24870">issues.joomla.org/tracker/joomla-cms/24870</a>.</sub>

File: administrator/components/com_users/View/Levels/HtmlView.php [hunk 3]
```diff
@@ -99,7 +91,6 @@ public function display($tpl = null)
 		}
 
 		$this->addToolbar();
-		$this->sidebar = HTMLHelper::_('sidebar.render');
```
Comment (Quy) [inline administrator/components/com_users/View/Levels/HtmlView.php:102]: Remove `use Joomla\CMS\HTML\HTMLHelper;`

File: administrator/components/com_users/View/Notes/HtmlView.php [hunk 4]
```diff
@@ -120,7 +112,6 @@ public function display($tpl = null)
 		}
 
 		$this->addToolbar();
-		$this->sidebar = HTMLHelper::_('sidebar.render');
```
Comment (Quy) [inline administrator/components/com_users/View/Notes/HtmlView.php:123]: Remove `use Joomla\CMS\HTML\HTMLHelper;`

File: administrator/components/com_users/View/Users/HtmlView.php [hunk 5]
```diff
@@ -121,8 +113,6 @@ public function display($tpl = null)
 		}
 
 		$this->addToolbar();
-		$this->sidebar = HTMLHelper::_('sidebar.render');
```
Comment (Quy) [inline administrator/components/com_users/View/Users/HtmlView.php:124]: Remove `use Joomla\CMS\HTML\HTMLHelper;`

File: administrator/components/com_users/View/Groups/HtmlView.php [hunk 6]
```diff
@@ -99,7 +91,6 @@ public function display($tpl = null)
 		}
 
 		$this->addToolbar();
-		$this->sidebar = HTMLHelper::_('sidebar.render');
```
Comment (Quy) [inline administrator/components/com_users/View/Groups/HtmlView.php]: Remove `use Joomla\CMS\HTML\HTMLHelper;`.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ccd331eab81f71683f6264312bb19aa79e004745<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24870">issues.joomla.org/tracker/joomla-cms/24870</a>.</sub>

**Comment by travisrisner** [comment]
I have tested this item :white_check_mark: successfully on ccd331e

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24870">issues.joomla.org/tracker/joomla-cms/24870</a>.</sub>

**Comment by roland-d** [comment]
Thank you.

**Comment by brianteeman** [comment]
thanks

**Pull request merged**